nvflare/apis/impl/study_manager.py

```diff
@@ -54,7 +60,7 @@ def create_study(self, study: Study) -> Tuple[Optional[Study], str]:
         Returns: updated study info (e.g. create_time is set)
 
         """
-        if self.get_study(study.name):
+        if study.name in self._existing_studies:
             return None, "Unable to create duplicated study name."
         if not study.participating_clients:
             return None, "Study has no participating clients."
```

Comment:
if we change the method name, maybe change this error msg as well -> `Unable to add duplicated study name.` ?